internal/support/notifications/config.go

```diff
@@ -14,7 +14,10 @@
  *******************************************************************************/
 package notifications
```

Comment:
While you're in here, please add an empty line between comment and package statement.